Makefile

```diff
@@ -30,3 +30,19 @@ test-carthage:
 # Tests if current branch ships a valid Cocoapods project.
 test-cocoapods:
 		@cd dependency-manager-tests/cocoapods && $(MAKE)
+
+bump:
+		@read -p "Enter version number: " version;  \
+		echo "// GENERATED FILE: Do not edit directly\n\ninternal let sdkVersion = \"$$version\"" > Sources/Datadog/Versioning.swift; \
+		sed "s/__DATADOG_VERSION__/$$version/g" DatadogSDK.podspec.src > DatadogSDK.podspec; \
+		sed "s/__DATADOG_VERSION__/$$version/g" DatadogSDKObjc.podspec.src > DatadogSDKObjc.podspec; \
+		git add . ; \
+		git commit -m "Bumped version to $$version"; \
+		git tag $$version; \
```

Comment:
Let's not add `tag` 🙌 - we do releases through GH > "New Release" UI and it provides the option to add tag + mark it as pre-release (plus a nice UI for crafting release notes 🙂). If we push tag from `make bump`, but have to fix something in podspecs for `make ship`, this will lead to inconsistency. Doing pre-release + tag through GH UI, then calling `make ship` and then changing from pre-release to regular release guarantees consistency.

Reply:
ok, removed that line